### DISCUSSION TITLE
[Question] How to enable Supavisor for self-hosted version

### DISCUSSION THREAD

**Discussion by nguyenthanhtien** [supabase/supabase#22057] [Questions]
I ran supabase with self-hosted version local, i ran in docker-compose, but i want ask how to can using [supavisor] for docker-compose at local. And [supavisor] is available for k8s helm chart version?

**Comment by BonsoirDiep** [comment]
Maybe help
https://www.nodejsauto.com/2024/07/ha-supabasetry-2-highly-available.html
